SQL/advanced.sql

```diff
@@ -383,7 +383,7 @@ CREATE VIEW booking_costs_today AS (
 STORED FUNCTIONS
 */
 
--- passing parameters
+/* #1: passing parameters */
 CREATE OR REPLACE PROCEDURE remove_member(
   member_id integer
```

Comment:
let's assume you have column named "member_id" in the table. How will the query look like? Does it read good?

Reply:
Suggesting using `_member_id` as the parameter?

Reply:
Otherwise in the case you described above we get PostgreSQL exception like "ambiguous name"

Reply:
right.